.changelog/1213.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:improvement
+internal/cli: Render new documentation subfields nicely
+builtin/aws/ecs: Render logging attributes as subfields as example
```

Comment:
can we do `plugin/aws-ecs` here? We did `builtin/` for one release but I think that was a mistake, early on we were doing plugin which I think is more useful to passers-by.

Reply:
Also I believe go-changelog expects separate entries for each improvement:

https://github.com/hashicorp/waypoint/tree/main/.changelog#multiple-entries

At least according to what Consuls docs mentioned. Seems noisey if true 🤔

Reply:
Oh, is this NOT the same as we do commit messages?

Reply:
Don't mind either way. I think plugins are the only weird one.

Reply:
@briancain Explained they're abstract user facing components.